infrastructure/kctf/base/Makefile

```diff
@@ -176,7 +192,6 @@ ${CLUSTER_GEN}/image-pushed: ${CLUSTER_GEN}/image-tagged
 	if [ "${PUSH_TARGET}" == "KIND" ]; then
```

Comment:
I think it will bite us at some point that we're using variables to change the behavior of non-phony targets. I.e. this is only ok since the variable is also implicitly part of the target name (since $(CLUSTER_GEN) will be different per config).
Maybe it would be less error-prone to test CLUSTER_GEN against a certain pattern instead?

Reply:
but anyway, this is not really related to this change in particular. LGTM!

Reply:
ok, 👍 not sure what you meant re: testing CLUSTER_GEN against a pattern, but we can figure this out offline